index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hello Microverse</title>
+</head>
+<body>
+    <h1 class="header">Hello Microverse</h1>
```

Comment:
I think your heading class can be named something other than "header". The header is a semantic tag you can use to group your head section.

Thank you😎

Reply:
Well done here 💯 this is very polite and it explains why you're requesting the change here 👍🏽 only that you need to add at least 2 comments in the review, you may use the same principle you used here. 😉

Reply:
Thank you for the advice😎

Reply:
You're welcome @Paulinelali but I think you need to perform the review again 😅 and add 2 inline comments.